src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/MetadataStoreRepositoryImpl.kt

```diff
@@ -129,6 +130,7 @@ class MetadataStoreRepositoryImpl(private val connectionSupplier: ConnectionSupp
             }
 
     override fun deleteAllReconciledRecords(deletedAccumulation: Int): Int {
+        logger.info("Beginning to delete reconciled records")
```

Comment:
Worth adding deletedAccumulation into the log?

Reply:
It's logged further down with all values: https://github.com/dwp/kafka-to-hbase-reconciliation/pull/47/files/285b043e00f3ffb3b1a5e4b1bb4a1d631b1468fc#diff-96fb4115f7061357e40fe341bf56c934187a43a1aa61ca842fcc9806791fe685R149